sdks/python/apache_beam/examples/inference/multi_language/last_word_prediction/src/main/java/org/MultiLangRunInference.java

```diff
@@ -0,0 +1,93 @@
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.io.IOException;
+import org.apache.beam.model.pipeline.v1.ExternalTransforms;
+import org.apache.beam.runners.core.construction.External;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.coders.RowCoder;
+import org.apache.beam.sdk.io.TextIO;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.options.Validation.Required;
+import org.apache.beam.sdk.schemas.Schema;
+import org.apache.beam.sdk.schemas.Schema.Field;
+import org.apache.beam.sdk.schemas.Schema.FieldType;
+import org.apache.beam.sdk.schemas.SchemaTranslation;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.util.ByteStringOutputStream;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.Row;
+import org.apache.beam.sdk.values.PDone;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MultiLangRunInference {
+    public interface MultiLangueageOptions extends PipelineOptions {
+
+        @Description("Path to an input file that contains labels and pixels to feed into the model")
+        @Required
+        String getInputFile();
+
+        void setInputFile(String value);
+
+        @Description("Path to an input file that contains labels and pixels to feed into the model")
+        @Required
+        String getOutputFile();
+
+        void setOutputFile(String value);
+    }
+
+    private static byte[] toStringPayloadBytes(String model) {
+        Row configRow = Row.withSchema(Schema.of(Field.of("model", FieldType.STRING)))
+                .withFieldValue("model", model)
+                .build();
+
+        ByteStringOutputStream outputStream = new ByteStringOutputStream();
+
+        try {
+            RowCoder.of(configRow.getSchema()).encode(configRow, outputStream);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        ExternalTransforms.ExternalConfigurationPayload payload = ExternalTransforms.ExternalConfigurationPayload
+                .newBuilder()
+                .setSchema(SchemaTranslation.schemaToProto(configRow.getSchema(), false))
+                .setPayload(outputStream.toByteString())
+                .build();
+        return payload.toByteArray();
+    }
```

Comment:
@chamikaramj there's an easier way to pass strings, right? Shouldn't this encoding happen automatically?

Reply:
Yeah, you should be able to just pass strings and that should be encoded (via UTF8) automatically.